archive/use-cases.html

```diff
@@ -139,6 +144,28 @@
      Other use cases may include bulk transfer of contacts cards, photos,
      and other data.
     </p>
+    <p class="note">
+      This use case is not supported in this version of the specification.
```

Comment:
Some places it is "not within scope" but here we talk about the spec - confusing